src/components/exit-this-page/index.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Exit this page
+description: Give users a way to quickly and safely exit a service, website or application.
+section: Components
+aliases:
+backlog_issue_id: 213
+layout: layout-pane.njk
+status: Experimental
+statusMessage: This component is currently experimental. <a class="govuk-link" href="#known-issues-and-gaps">You'll need to do your own research</a> to decide whether to add this component to your service.
+---
+{% from "_example.njk" import example %}
+
+Give users a way to quickly and safely exit a service, website or application.
+
+For service journeys, you must use this component with the pattern to help a user [exit a page quickly](/patterns/exit-a-page-quickly/).
+
+{{ example({group: "components", item: "exit-this-page", example: "default", html: true, nunjucks: true, open: false, size: "s"}) }}
+
+## When to use this component
+
+Use the component on pages with sensitive information that could:
+- put someone at risk of abuse or retaliation
+- reveal someone’s plans to avoid or escape from harm
+
+For example, when a potential victim is using a service to help them leave a domestic abuser.
+
+You can use this component on either:
+- all the pages in a service
+- parts of the journey with sensitive information
+
+You can also use this component for standalone content pages, such as dashboards and guidance.
+
+## When not to use this component
+
+Do not use this component if the service or content is unlikely to put a user at risk. See the [Exit a page quickly](/patterns/exit-a-page-quickly/) pattern for examples of at-risk and sensitive topics.
+
+Keep in mind that seeing this component might discourage certain users from using your service. If the user does not identify themselves as being at risk, they might see the button on a service and decide it’s not relevant to them.
+
+## How it works
+
+There are 2 ways to use the exit this page component. You can use HTML or, if you are using Nunjucks or the GOV.UK Prototype Kit, you can use the Nunjucks macro.
+
+## Adding a button
+
+Position the component at the top of your page, above the beginning of the grid (`<div class="govuk-grid-row">`) but still within the width container (`<div class="govuk-width-container">`).
+
+### Choosing a different website to load
+
+When activated, the component by default will take the user to the [homepage of BBC Weather](https://www.bbc.co.uk/weather). You can change this if there’s a more appropriate site for your users.
+
+Avoid websites that might show personalised pages (such as frequently visited, last visited, or suggested links), as this content could put the user at risk.
+
+## Consider what to do with user session data
+
+You’ll need to decide if your service will store the user’s session data after they’ve activated the Exit this page component and left your service.
+
+We recommend that services clear the user’s session data once they’ve activated Exit this page.
```

Comment:
Another non-blocker: I'm wondering there's an opportunity go into more detail here or if that would muddy the waters. An example of what I'm thinking:

```suggestion
We recommend that services clear the user’s session data once they’ve activated Exit this page. An example of how to do this in your service could be to change the default button URL to a page in your service which when visited, clears the user's session data then automatically redirects to an inconspicuous website.
```